front/src/features/tables/helpers/tagArrayManagement.ts

```diff
@@ -0,0 +1,30 @@
+import { Tag } from "@prisma/client";
+
+export const isTagInArray = (tags: Tag[], tagId: string): boolean => {
+    return tags.some((tag) => tag.id === tagId);
+};
+
+export const findTagById = (tags: Tag[], tagId: string): Tag => {
+    const tag = tags.find((tag) => tag.id === tagId);
+    if (tag === undefined) {
+        throw new Error(
+            `Failed to find tag of id ${tagId} among the given tags.`,
+        );
+    }
+    return tag;
+};
+
+export const concatAndSortTags = (tags: Tag[], newTag: Tag): Tag[] => {
+    const newTagArray = [...tags, newTag];
+    return newTagArray.sort((tagA, tagB) => {
+        return compareStrings(tagA.name, tagB.name);
+    });
+};
+
+const compareStrings = (stringA: string, stringB: string): number => {
```

Comment:
hmm I thought sort basically did this by default, interesting you needed to define a sort function yourself since its a basic string sort

Reply:
Can't you just do ```return newTagArray.sort(tagA.toLowerCase(), tagB.toLowerCase());```

Reply:
I did a little research and found a built-in function that compares string based on alphabetical order. New commit incoming!